Cargo.toml

```diff
@@ -17,3 +17,7 @@ debug = true
 codegen-units = 1
 inherits = "release"
 lto = "yes"
+
+[patch.crates-io]
```

Comment:
Note the explicit patch to align with the containers-image-proxy change here, need to coordinate moving these in tandem.